JS/app.js

```diff
@@ -25,19 +26,49 @@ photoArray.push(new Photo('img/kitten12.jpg'));
 photoArray.push(new Photo('img/kitten13.jpg'));
 photoArray.push(new Photo('img/kitten14.jpg'));
 
-//etc
+var leftVotes;
+var rightVotes;
```

Comment:
I would suggest putting these 'module-level' variables at the top of your script, so anyone reading will know what variables are in play within the code. It will help you as well, with one single place for the module level vars to live.